docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc

```diff
@@ -62,17 +62,13 @@ Full List of Resources:
 12: vlan40	inet 172.17.4.24/24 brd 172.17.4.255 scope global vlan40\   	valid_lft forever preferred_lft forever
 ----
 +
-* vlan30 represents the Storage Network, where the new RGW instances should be
-started on the {CephCluster} nodes.
-* br-ex represents the External Network, which is where in the current
-environment, haproxy has the frontend Virtual IP (VIP) assigned.
+<1> `br-ex` represents the External Network, where in the current
+environment, HAProxy has the front-end Virtual IP (VIP) assigned.
+<2> `vlan30` represents the Storage Network, where the new RGW instances should be started on the {CephCluster} nodes.
 
-. Identify the network that you previously had in haproxy and propagate it through
-{OpenStackPreviousInstaller} to the {CephCluster} nodes. This network is used to reserve a new VIP
-that is owned by {Ceph} and used as the entry point for the RGW service.
+. Identify the network that you previously had in HAProxy and propagate it through {OpenStackPreviousInstaller} to the {CephCluster} nodes. This network is used to reserve a new VIP that is owned by {Ceph} and used as the entry point for the RGW service.
```

Comment:
could make this less passive: "This network is used to reserve a new VIP that is owned by {Ceph} and used as the entry point for the RGW service"
Maybe:
"Use this network to reserve a new VIP that is owned by {Ceph} as the entry point for the RGW service."

Is it VIP or vIP?